src/common/telemetry/src/logging.rs

```diff
@@ -234,7 +234,13 @@ pub fn init_global_logging(
                     opentelemetry_otlp::new_exporter().tonic().with_endpoint(
                         opts.otlp_endpoint
                             .as_ref()
-                            .map(|e| format!("http://{}", e))
+                            .map(|e| {
```

Comment:
question: Do we have `https` scenarios?

Reply:
I think we should leave the choice to the user